### DISCUSSION TITLE
v1.1

### DISCUSSION THREAD

**Discussion by rabbibotton** [rabbibotton/clog#103] [General]
This new release of CLOG includes minor fixes and documentation in addition to the CLOG Builder v1.0 a full featured extensible rapid visual development tool similar to VB/Delphi for creating CLOG Panels for use as components on web pages or in CLOG-GUI projects, or CLOG Pages full web pages with built in support for w3css and bootstrap 5.

<hr /><em>This discussion was created from the release <a href='https://github.com/rabbibotton/clog/releases/tag/v1.1'>v1.1</a>.</em>

**Comment by aykaramba** [comment]
Awesome!
